options/middleware.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Comment:
this file might more appropriately be named something more general in the context of attaching additional annotations to methods.